frontend/src/types/validate-analytics.ts

```diff
@@ -0,0 +1,70 @@
+import type { Events } from "~/types/analytics"
+
+import type { UnionToIntersection } from "type-fest"
+
+type ReservedPropNames =
```

Comment:
If we break these into WindowPropNames and IsomorphicPropNames, they can be used as type hints for [`windowProps`](https://github.com/WordPress/openverse/pull/1125/files#diff-d6fe83def5944105827719ecdad03ddd3b8b989855454f17c241c571a0c1f4c1L38) and [`isomorphicProps`]((https://github.com/WordPress/openverse/pull/1125/files#diff-d6fe83def5944105827719ecdad03ddd3b8b989855454f17c241c571a0c1f4c1L19), making it easier to keep this list in sync with the actual props.